GitUI/UserControls/FileStatusList.cs

```diff
@@ -570,7 +571,7 @@ public IEnumerable<string> SelectedItemParents
             {
                 return FileStatusListView.SelectedItems.Cast<ListViewItem>()
                     .Where(i => i.Group != null)
-                    .Select(i => (string)i.Group.Tag);
+                    .Select(i => ((GitRevision)i.Group.Tag)?.Guid);
```

Comment:
I am not sure this is completely safe cast.
Perhaps the following is better?

```
                return FileStatusListView.SelectedItems.Cast<ListViewItem>()
                     .Where(i => i.Group != null && i.Group.Tag as GitRevision != null)
                     .Select(i => ((GitRevision)i.Group.Tag).Guid);
```

Reply:
Can change all casts for Tag to be something like this